components/builder-protocol/protocols/sessionsrv.proto

```diff
@@ -94,27 +95,28 @@ message Session {
   optional uint64 id = 1;
   optional string email = 2;
   optional string name = 3;
+  // base64 encoded binary representation of SessionToken
   optional string token = 4;
   optional uint32 flags = 5;
-}
-
-// This can be deleted
-message SessionToken {
-  optional string token = 1;
-  optional uint64 owner_id = 2;
-  optional OAuthProvider provider = 3;
+  optional string oauth_token = 6;
 }
 
 message SessionCreate {
   optional string token = 1;
-  optional uint64 extern_id = 2;
+  optional uint32 extern_id = 2;
```

Comment:
This change is fine, we don't want the old authentication flow to work anyway at all. It was a security issue